manifests/dev/vsphere-7.0u2/guestcluster/1.18/pvcsi.yaml

```diff
@@ -362,8 +405,21 @@ data:
     [FeatureStatesConfig]
     name = "csi-feature-states"
     namespace = "{{ .PVCSINamespace }}"
```

Comment:
both `FeatureStatesConfig` and `InternalFeatureStatesConfig` uses same variable `PVCSINamespace`?
I guess one is for supervisor and another is for a guest?

Reply:
Yes! InternalFeatureStatesConfig is specific to guest cluster and FeatureStatesConfig was introduced in 7.0U1 as a replica of the supervisor FSS.